test/parallel/test-https-localaddress-bind-error.js

```diff
@@ -28,9 +28,11 @@ const assert = require('assert');
 const fs = require('fs');
 const https = require('https');
 
+const fixtures = require('../common/fixtures');
+
 const options = {
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fs.readFileSync(`${fixtures.fixturesDir}/keys/agent1-key.pem`),
```

Comment:
This could be using method `fixtures.readKey` instead of `fs.readFileSync` to get the keys.

Reply:
Correct, the proper change here would be:

```js
const options = {
  key: fixtures.readKey('agent1-key.pem'),
  cert: fixtures.readKey('agent1-cert')
}
```